linkerd/app/outbound/src/http/peer_proxy_errors.rs

```diff
@@ -41,146 +37,89 @@ where
     }
 }
 
-impl<S, A, B> Service<Request<A>> for PeerProxyErrors<S>
+impl<S, A, B> svc::Service<http::Request<A>> for PeerProxyErrors<S>
 where
-    S: Service<Request<A>, Response = Response<B>>,
+    S: svc::Service<http::Request<A>, Response = http::Response<B>>,
     S::Response: Send,
     S::Future: Send + 'static,
-    A: Send + 'static,
     B: Send + 'static,
 {
     type Response = S::Response;
     type Error = S::Error;
     type Future = Pin<Box<dyn Future<Output = Result<S::Response, S::Error>> + Send + 'static>>;
 
+    #[inline]
```

Comment:
We try to use `#[inline]` with trivial passthru implementations, especially for `poll_ready`.  Basically: there's no use seeing this scope in backtraces.